### PR TITLE
feat: add  arg to standalone start command

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add `--data-home` / `-d` arg to `greptime [standalone|metasrv] start` command. One can change the standalone, datanode and metasrv instance's working directory from command line from now on.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by killme2008 — COMMENTED** [review]
We need to support it in datanode too.

File: src/cmd/src/metasrv.rs [hunk 1]
```diff
@@ -100,6 +100,9 @@ struct StartCommand {
     http_timeout: Option<u64>,
     #[clap(long, default_value = "GREPTIMEDB_METASRV")]
     env_prefix: String,
+    /// The working home directory of this metasrv instance.
+    #[clap(short('d'), long)]
```
Comment (killme2008) [inline src/cmd/src/metasrv.rs:104]: `-d` often means running in the background. Maybe don't use any short names or change it to `-h` would be better.

**Review by killme2008 — APPROVED** [review]
LGTM 👍

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**